libs/core/src/lib/notification/notification-service/notification.service.ts

```diff
@@ -4,37 +4,29 @@ import { NotificationContainer } from '../notification-utils/notification-contai
 import { NotificationConfig } from '../notification-utils/notification-config';
 import { NotificationRef } from '../notification-utils/notification-ref';
 import { DynamicComponentService } from '../../utils/dynamic-component/dynamic-component.service';
-import { NotificationGroupComponent } from '../notification-group/notification-group.component';
-import { NotificationDefault } from '../notification-utils/notification-default';
 
 @Injectable()
 export class NotificationService {
     public notifications: {
         notificationComponent: ComponentRef<NotificationComponent>;
-        notificationGroup?: ComponentRef<NotificationGroupComponent>;
     }[] = [];
+    
     public containerRef: ComponentRef<NotificationContainer>;
 
     constructor(private dynamicComponentService: DynamicComponentService) {}
 
     /**
-     * Opens an alert component with a content of type TemplateRef, Component Type or Configuration Object
-     * @param content Content of the alert component, or NotificationDefault object.
+     * Opens a notification component with a content of type TemplateRef or Component Type
      * @param notificationConfig Configuration of the notification component.
-     * @param notificationGroup Configuration of the notification component.
      */
     public open(
-        content: TemplateRef<any> | Type<any> | NotificationDefault,
-        notificationConfig: NotificationConfig = new NotificationConfig(),
-        notificationGroup?: ComponentRef<NotificationGroupComponent>
+        content: TemplateRef<any> | Type<any>,
```

Comment:
"content" parameter was removed from documentation while still present in argument list

Reply:
Also noticed that isAnyOpened method is unused anywhere, and maybe should be removed, or at least added to tests?

Reply:
`isAnyOpened()` was added in the previous implementation of the component in case the user wants to check if there are open notifications before reacting to some event (for example, not allowing the user to change the page, etc.)

Reply:
> "content" parameter was removed from documentation while still present in argument list

I'm not sure I understand your comment.

Reply:
The content here refers to a template or a component. In the documentation we have 2 examples -  "Template as content" and "Component as content".

Reply:
I meant this line: @param content Content of the alert component, or NotificationDefault object.
It was removed from the JsDoc, while the parameter wasn't

Reply:
gotcha. Put back the content param